pkg/virt-launcher/virtwrap/network/podinterface.go

```diff
@@ -177,7 +177,7 @@ func (l *podNICImpl) PlugPhase1(vmi *v1.VirtualMachineInstance, iface *v1.Interf
 	}
 
 	pidStr := fmt.Sprintf("%d", pid)
-	isExist, err := bindMechanism.loadCachedInterface(pidStr, iface.Name)
+	isExist, err := bindMechanism.loadCachedInterface(pidStr)
```

Comment:
I think we can remove `pidStr` from `loadCacheInterface` but I understand that it need major rework since it will change the BindMechanism interface, can we do it at a follow up PR ?

Reply:
That is the objective - once I can refactor the caching mechanism between both processes, I plan to remove it.

The reasons why that particular parameter was left behind are explained in the commit msgs. Let me know if they're not clear.

Reply:
@qinqon I think I did this in https://github.com/kubevirt/kubevirt/pull/5059/commits/280a78321bdad628396d8dedef6454faf2e1265b, which is now possible after the new caching mechanism set in place.

Your thoughts ?